zerver/lib/validator.py

```diff
@@ -178,6 +178,13 @@ def validator(var_name: str, val: object) -> int:
     return validator
 
 
+def check_int_param_in(var_name: str, val: int, possible_values: List[int]) -> int:
+    n = check_int(var_name, val)
+    if n not in possible_values:
+        raise ValueError(_("Invalid {var_name}").format(var_name=var_name))
+    return n
```

Comment:
I think the intent of the Pydantic system is that we not end up using anything in this file. Note also this file is intended to always raise `ValidationError`. You can add new Pydantic validators in `zerver/lib/typed_endpoint_validators.py`; this file will eventually be deleted when we finish the migration and shouldn't be polluted with parts of the new system.